diff --git src/agent/rustjail/src/mount.rs src/agent/rustjail/src/mount.rs
index 8c021cc..b85e80d 100644
--- src/agent/rustjail/src/mount.rs
+++ src/agent/rustjail/src/mount.rs
@@ -34,6 +34,7 @@ use crate::log_child;
 // Info reveals information about a particular mounted filesystem. This
 // struct is populated from the content in the /proc/<pid>/mountinfo file.
 #[derive(std::fmt::Debug)]
+#[allow(dead_code)]
 pub struct Info {
     id: i32,
     parent: i32,
