diff --git src/agent/src/network.rs src/agent/src/network.rs
index 244673d..c12ea75 100644
--- src/agent/src/network.rs
+++ src/agent/src/network.rs
@@ -16,6 +16,7 @@ const GUEST_DNS_FILE: &str = "/etc/resolv.conf";
 // Network fully describes a sandbox network with its interfaces, routes and dns
 // related information.
 #[derive(Debug, Default)]
+#[allow(dead_code)]
 pub struct Network {
     ifaces: HashMap<String, Interface>,
     routes: Vec<Route>,
